src/site/includes/vet_center_hours.liquid

```diff
@@ -0,0 +1,4 @@
+<script nonce="**CSP_NONCE**" type="text/javascript">
+    window.vetCenterHours = {{ vetCenterHours | json }};
+</script>
+<div data-widget-type="vet-center-hours"></div>
```

Comment:
Instead of passing the vetCenterHours data via the `window` global, how about passing it as a data attribuge on the div, e.g.

```suggestion
<div data-widget-type="vet-center-hours"
        data-hours="{{ vetCenterHours | json }}"
></div>
```

We might not even need this file if we take that approach.